pmdarima/arima/arima.py

```diff
@@ -587,15 +579,16 @@ def predict_in_sample(self, exogenous=None, start=None,
         else:
             preds = results_wrapper.predict(
                 exog=exogenous, start=start,
-                end=end, dynamic=dynamic)
+                end=end, dynamic=dynamic, **kwargs)
 
             # We only have to un-integrate if there was any differencing
+            # TODO: prob don't need to since this occurs in the predict code
             # (i.e., ARIMA vs. ARMA)
-            if d > 0:
-                preds = _unintegrate_differenced_preds(
-                    preds=preds,
-                    results_wrapper=results_wrapper,
-                    d=d)
+            # if d > 0:
+            #     preds = _unintegrate_differenced_preds(
+            #         preds=preds,
+            #         results_wrapper=results_wrapper,
+            #         d=d)
```

Comment:
This can really disappear (probably). But I left it until we are certain this is correct.